### PR TITLE
Ik/move cli tests

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run ote-test

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote-test

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote-test

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote-test

File: external/anomaly/init_venv.sh [hunk 1]
```diff
@@ -112,7 +107,8 @@ fi
 pip install -r requirements.txt
 pip install -e .
 
-pip install -e $OTE_SDK_PATH  || exit 1
+# Install OTE SDK
+pip install -e ../../ote_sdk/ || exit 1
```
Comment (LeonidBeynenson) [inline external/anomaly/init_venv.sh:111]: Ilya, sorry, do not use relative paths

File: external/anomaly/init_venv.sh [hunk 2]
```diff
@@ -112,7 +107,8 @@ fi
 pip install -r requirements.txt
 pip install -e .
```
Comment (LeonidBeynenson) [inline external/anomaly/init_venv.sh]: Ilya, please, insert `|| exit 1` here

**Review by LeonidBeynenson — CHANGES_REQUESTED** [review]

File: external/anomaly/init_venv.sh (hunk 2 above)
Comment (Ilya-Krylov) [reply external/anomaly/init_venv.sh]: fixed

**Comment by Ilya-Krylov** [comment]
run ote-test

**Pull request merged**